src/consumer/component/consumer-component.js

```diff
@@ -1064,8 +1089,12 @@ export default class Component {
       if (dependency) this.dependencies.add(dependency);
       else {
         const devDependency = componentFromModel.devDependencies.getByIdStr(id);
-        if (!devDependency) throw new Error(`copyDependenciesFromModel unable to find dependency ${id} in the model`);
-        this.devDependencies.add(dependency);
+        if (devDependency) this.devDependencies.add(dependency);
```

Comment:
Consider adding a curly brace when it's an if else statement

Reply:
Will be fixed in the next commit.
Was planning to refactor the entire section.